core/aws-core/src/main/resources/software/amazon/awssdk/global/handlers/execution.interceptors

```diff
@@ -0,0 +1 @@
+software.amazon.awssdk.awscore.eventstream.EventStreamInitialRequestInterceptor
```

Comment:
I think we need to add `reflect-config.json ` file in this module for this interceptor.  That should fix the `aws-sdk-java-v2-native-image-test` build

Example: https://github.com/aws/aws-sdk-java-v2/blob/master/core/sdk-core/src/main/resources/META-INF/native-image/software.amazon.awssdk/sdk-core/reflect-config.json

Reply:
OK, I'll try that.